buildenv/jenkins/clang_format/clangFormatCheck.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash -f
+###############################################################################
+# Copyright IBM Corp. and others 2025
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+# or the Apache License, Version 2.0 which accompanies this distribution and
+# is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following
+# Secondary Licenses when the conditions for such availability set
+# forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+# General Public License, version 2 with the GNU Classpath
+# Exception [1] and GNU General Public License, version 2 with the
+# OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] https://openjdk.org/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+###############################################################################
+
+allFiles=$(git diff -C --diff-filter=ACM --name-only origin/master HEAD --)
+if [ x"$allFiles" = x ] ; then
+    echo "There are no files to check for code formatting."
+else
+    lines='-----------------------------------'
+    badFiles=
+    for file in $allFiles ; do
```

Comment:
I point out that this won't work if any file or directory names contain spaces. If ever we need to support that, this can change to something like:
```sh
    git diff -C --diff-filter=ACM --name-only origin/master HEAD -- \
    | while read -r file ; do
```